app/common/app_config.js

```diff
@@ -75,6 +81,7 @@ class TasksConfig extends AppConfig {
 }
 
 module.exports = {
+    CONFIG,
```

Comment:
Maybe `FIELDS` (or `fields`) instead of `CONFIG` (mainly to avoid AppConfig.CONFIG which is a bit redundant

Reply:
Yup, makes sense, I'll update to `FIELDS` instead.